template/azurePipelines/manifests/.pipelines/azure-kubernetes-service.yaml

```diff
@@ -0,0 +1,60 @@
+# Azure Kubernetes Service pipeline
+# Build and push image to Azure Container Registry; Deploy to Azure Kubernetes Service cluster
+
+variables:
+  armServiceConnection: {{ARM_SERVICE_CONNECTION}}
+  azureContainerRegistry: {{AZURE_CONTAINER_REGISTRY}}
+  containerName: {{CONTAINER_NAME}}
+  clusterRg: {{CLUSTER_RESOURCE_GROUP}}
+  acrRg: {{ACR_RESOURCE_GROUP}}
+  clusterName: {{CLUSTER_NAME}}
+  manifestPath: {{MANIFEST_PATH}}
+  namespace: {{NAMESPACE}}
+  tag: "$(Build.BuildId)"
+  vmImageName: "ubuntu-latest"
+
+name: {{PIPELINE_NAME}}
+
+trigger:
+  - main

```

Comment:
trigger is hardcoded here when it's not in kustomize

Reply:
Thanks for catching that, changed it into a variable